src/cellviz.cpp

```
@@ -41,44 +44,47 @@ int main(int argc, char *argv[]) {
 
     double sum = 0;
     double mean = 0;
-    for (const auto& price : prices) {
-        sum += price;
-    }
+    for (const auto& price : prices) sum += price; //
     mean = sum / count;
 
     int size = count / 2;
-    cout << "Size: " << size << endl;
 
```

cppcheck

[performance] Variable 'species' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)